### DISCUSSION TITLE
After migration of bucket, simply change bucket URL for juicefs mount ?

### DISCUSSION THREAD

**Discussion by BeatTakeshi** [juicedata/juicefs#3163] [Q&A] [answered]
Hi!

If I want to switch S3-Bucket and use some other (cloud) tool to migrate all data, can I simply change the Bucket-URL in an existing JuiceFS formatted Filesystem ? And if so, where would I need to do this ? Or do I need to "re-format" (found some mention of "updating" a format, but no description what this is or does exactly) 

Background: Want to create a new bucket on Backblaze in Europe-Region and use Backblaze "sync" to move all data, how do I then switch an existing juicefs mount to the new bucket ?

Thanks in advance and BR!

**Comment by zhijian-pro** [comment, marked as answer]
The `juicefs config` command can help you.
